api/service/explorer/service.go

```diff
@@ -258,6 +266,52 @@ func (s *Service) GetExplorerTransaction(w http.ResponseWriter, r *http.Request)
 	data.TX = *tx
 }
 
+// GetCommittee servers /comittee end-point.
+func (s *Service) GetCommittee(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	shardIDRead := r.FormValue("shard_id")
+	epochRead := r.FormValue("epoch")
+	shardID := uint64(0)
+	epoch := uint64(0)
+	var err error
+	if shardIDRead != "" {
+		shardID, err = strconv.ParseUint(shardIDRead, 10, 32)
+		if err != nil {
+			utils.Logger().Warn().Err(err).Msg("cannot read shard id")
```

Comment:
Return a 400 error; do not proceed and use uninitialized or partially-initialized `shardID`.